app/logic/Mail/ActiveSync/ActiveSyncError.ts

```diff
@@ -1,10 +1,12 @@
 import type { Account } from "../../Abstract/Account";
+import type { Calendar } from "../../Calendar/Calendar";
+import type { Addressbook } from "../../Contacts/Addressbook";
 import { gt } from "../../../l10n/l10n";
 
 export class ActiveSyncError extends Error {
   type: string;
   code: string;
-  constructor(aCommand: string, aStatus: string, account: Account) {
+  constructor(aCommand: string, aStatus: string, account: Account | Calendar | Addressbook) {
```

Comment:
Line 231 of `ActiveSyncEvent.ts` passes `this.calendar.account` but you probably want to change that to `this.calendar`.